### DISCUSSION TITLE
Understanding Slugs / admin UI names?

### DISCUSSION THREAD

**Discussion by Rar9** [payloadcms/payload#1279] [Q&A] [answered]
A Slug is the unique identifying part of a web address, typically at the end of the URL and its recommend to avoid special characters like commas, semicolons, colons, spaces, quotes. 

![unsafe-characters-url.jpg](https://user-images.githubusercontent.com/2239608/196773587-9e287036-7436-43e0-b9b9-703fcb94f1e0.jpg)

From my understanding the Admin UI collection Names are generated from slugs. 

If I require a nicer admin UI Name, I would need to currently break the Slug suggestion by using a Space when using 2 Words instead of a dash/underscore.

![Screenshot_20221019_201534.jpg](https://user-images.githubusercontent.com/2239608/196773929-3bd28a72-93c4-4157-8ce0-b6f6351bbd36.jpg)

From Demo Slug code spaces would still produce correct urls. 

`const format = (val: string): string => val.replace(/ /g, '-').replace(/[^\w-]+/g, '').toLowerCase();`

Or did I oversee something that the admin UI Name can be specified somewhere? 

P. S. Perhaps adding for Admin UI a word wrap to get  2x lines could be added, as we still have enough space in our boxes.

**Comment by jmikrut** [comment, marked as answer]
Hey @Rar9 — you can specify nicer names using `labels: { singular: 'Page', plural: 'Pages' }` on Collections. But by default we use the `name` if you don't add more specificity. You can check the docs for more information here!

**Reply by Rar9** [reply]
Thanks for the fast response .... 

I searched in docs for slug / label / name/admin but this didn't show up.
